databaseQueries/certificateTemplates.js

```
@@ -94,4 +94,22 @@ module.exports = class CertificateTemplates {
+	 * @returns {Promise<Object>} - MongoDB deleteMany result containing deleted count.
+	 */
+	static deleteCertificateTemplate(filter) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				let deleteDocuments = await database.models.certificateTemplates.deleteMany(filter)
+
+				return resolve(deleteDocuments)
+			} catch (error) {
+				return reject(error)
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)